UI/obs-app.cpp

```diff
@@ -551,7 +551,9 @@ static bool MakeUserDirs()
 		return false;
 	if (!do_mkdir(path))
 		return false;
+#endif
 
+#if defined(_WIN32) || defined(WHATSNEW_ENABLED)
```

Comment:
With `WHATSNEW_ENABLED` treated as a global define (see above), the specific check for `_WIN32` could be removed.